src/components/MessageList/VirtualizedMessageList.js

```diff
@@ -125,27 +152,14 @@ const VirtualizedMessageList = ({
         totalCount={messages.length}
         overscan={overscan}
         followOutput={true}
```

Comment:
`alignToBottom` seems nice, but it's not consistent with our other messagelists?

Reply:
Also, `smooth` would be cool in most cases but the virtuoso docs mention that it may not be able to keep up with many messages coming in quickly. Might be risky for livestream customers?

Reply:
we can put in the default and allow customers to override them if it's not working for them